### PR TITLE
Add validations

### PR DESCRIPTION
**Pull request by elyor-doniyorov**
**In this branch 👇** 

`add validations using JavaScript;`

### CODE REVIEW

File: form.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+
+  // title validation
+  if(jobTitle === "") {
+    alert("enter the job title");
+    return false;
+  }
+
+  if(!isNaN(jobTitle)) {
+    alert("job title should be in letters");
```
Comment (OneSkool) [inline form.js:24]: Collect errors in a single array and then display error message

File: form.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  // let country = document.forms["myform"]["country"].value;
+  let remotePolicy = document.forms["myform"]["remotePolicy"].value;
+  let sponsorVisa = document.forms["myform"]["sponsorVisa"].value;
+  let salaryMin = document.forms["myform"]["salaryMin"].value;
+  let salaryMax = document.forms["myform"]["salaryMax"].value;
+  let expiresAt = document.forms["myform"]["expiresAt"].value;
+
+
+  // title validation
+  if(jobTitle === "") {
```
[the remark this answers is not in the document]
Comment (elyor-doniyorov) [reply form.js:18]: done

File: form.html [hunk 3]
```diff
@@ -1,123 +1,94 @@
...
+    </div>
 
-    <tr>
-      <td>jobStatus</td>
-      <td>Active<input type="radio" name="jobStatus" value="active" />
-        Not Active<input type="radio" name="jobStatus" value="Not Active" />
-      </td>
-    </tr>
+    <div>
+      <label for="jobStatus">jobStatus</label>
```
Comment (schohan) [inline form.html:50]: Labels should be named for users. So 'jobStatus' should be 'Job Status'.

**Review by schohan — CHANGES_REQUESTED** [review]
Where are error messages displayed? I would like to see it running on your laptop in our next meeting but make the requested changes.
In addition to this, try loading Form fields with JSON data as it would come from a backend API. Use a simple javascript function for it:

 `function load() {
    return {
       title: "some title",
       description: "description info",
       ....
    }
}`

**Comment by elyor-doniyorov** [comment]
> Where are error messages displayed? I would like to see it running on your laptop in our next meeting but make the requested changes. In addition to this, try loading Form fields with JSON data as it would come from a backend API. Use a simple javascript function for it:
> 
> `function load() { return { title: "some title", description: "description info", .... } }`

**Pull request closed without merging**